### PR TITLE
Json rdd to OrientDB export

### PR DESCRIPTION
**Pull request by szoio**
Created the ability to save from Json to OrientDB in Spark

### CODE REVIEW

File: spark-orientdb-connector/src/test/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctionsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  var rddTest: RDD[String] = null
+  var database: ODatabaseDocumentTx = null
+
+  override def beforeAll(): Unit = {
+    defaultSparkConf.set("spark.orientdb.dbname", dbname)
+    initSparkConf(defaultSparkConf)
+    rddTest = createRDD
+    database = createDB
+    rddTest.saveJsonToOrient("class_json_ins_test")
```
Comment (mourednik) [inline spark-orientdb-connector/src/test/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctionsSpec.scala]: This line really should be in the actual test.

File: spark-orientdb-connector/src/main/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctions.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class ClassJsonRDDFunctions(rdd: RDD[String]) extends Serializable with Logging {
+  /**
+    * Saves an instance of [[org.apache.spark.rdd.RDD RDD]] into an OrientDB class.
+    * @param myClass the OrientDB class to save in
+    */
+  /*
+	 * Prerequisites:
+	 *  -> Input class must have been created on OrientDB
+	 *  -> rdd must be composed of strings in Json format
+	 */ def saveJsonToOrient(myClass: String)(implicit connector: OrientDBConnector = OrientDBConnector(rdd.sparkContext.getConf)): Unit = {
```
Comment (mourednik) [inline spark-orientdb-connector/src/main/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctions.scala]: Need to run Scalariform

**Comment by mourednik** [comment]
Looks good. Just needs a couple of cleanups

File: spark-orientdb-connector/src/test/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctionsSpec.scala (hunk 1 above)
Comment (szoio) [inline spark-orientdb-connector/src/test/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctionsSpec.scala]: Agreed - this is exactly how they've done it in their test cases in `ClassRDDFunctionsSpec`.
I suppose I'm perpetuating the error by sticking to their flawed pattern.
I'll move it.

File: spark-orientdb-connector/src/main/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctions.scala (hunk 2 above)
Comment (szoio) [inline spark-orientdb-connector/src/main/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctions.scala]: I'm so used to having the plugin do this for me.

**Pull request merged**